README.md

```diff
@@ -1,75 +1,67 @@
-by defining a font library in the themename.libraries.yml file.
-7. If desired, add button mixins in src/scss/utilities/mixins/_buttons.scss with your button styles and then reference them and use them in src/scss/theme/_button.scss.
-8. If desired, add cta and link mixins in src/scss/utilities/mixins/_cta.scss with your button styles and then reference them and use them in src/scss/theme/_cta.scss.
-9. If you want to skin or theme a component, like a paragraph, create a _mycomponent.scss file in src/scss/components folder. Consider using a subfolder like 'cards' or 'banners' if applicable.
+
+Install this theme like any other theme. [See Drupal Documentation](https://www.drupal.org/docs/8/extending-drupal-8/installing-themes)
+
+This theme depends on the [`Components`](https://www.drupal.org/project/components) module to register template
+namespaces. If downloading this theme via composer, the module is already downloaded and simply needs to be enabled
+before the theme can be enabled.
+
+If using this theme in conjunction with configuration management and doing site install, a patch is included in the
+composer.json. Ensure that your project's root composer.json has [patching enabled](https://github.com/cweagans/composer-patches#allowing-patches-to-be-applied-from-dependencies)
+or download and patch Drupal Core manually.
 
 Configuration
 ---
 
-Nothing special needed. Install, enable, and set as the default active theme.
+Nothing special needed. Install, enable, and set as the default active theme. Once you have the theme enabled, be sure to check out the appearance settings for additional configuration items.
+
+
+Troubleshooting
+---
+
+If you are experiencing issues with this theme try disabling and clear the cache first. If you are still experiencing issues try posting an issue on the GitHub issues page.
+
+This theme is not intended to be used as an administration theme. Please use something else for that.
 
 Developer
 ---
 
-If you wish to develop on this theme you will most likely need to compile some new css. Please use the sass structure provided and compile with the sass compiler packaged in this theme. To install:
+Development tools: To build and compile the CSS, Javascript, and Image assets required to make this theme great, you will need to have npm and nvm installed.
+Navigate to the root of the theme then.
 
+Run:
 ```
+nvm use
 npm install
 ```
-After you've made a change you want to see processed, you can run:
+
+This project uses webpack to assemble the assets for this theme. To compile sass, javascript and push all assets in to place:
+
+Run:
 ```
-npm run build/yarn build
+npm run build
 ```
-This will process scss, js, and asset files, preparing them from the src directory to the dist directory.
 
+Or for development you can watch for changes and compile automatically with:
 ```
-npm run watch/yarn watch
+npm run watch
 ```
-This will watch the scss files and compile them upon saving.
-
-GitPod
```

Comment:
The GitPod stuff works super well (it is how we reviewed your subtheme changes!) Maybe we should keep this in here?